config/admin-functions/create-machine-cert.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+: "${VX_CONFIG_ROOT:="/vx/config"}"
+: "${VX_METADATA_ROOT:="/vx/code"}"
+
+MACHINE_CERT_PATH="${VX_CONFIG_ROOT}/vx-${VX_MACHINE_TYPE}-cert.pem"
+if [[ "${VX_MACHINE_TYPE}" == "admin" ]]; then
+    MACHINE_CERT_PATH="${VX_CONFIG_ROOT}/vx-${VX_MACHINE_TYPE}-cert-authority-cert.pem"
+fi
+MACHINE_PRIVATE_KEY_PATH="${VX_CONFIG_ROOT}/vx-${VX_MACHINE_TYPE}-private-key.pem"
+USB_CERTS_DIRECTORY="/media/vx/usb-drive/certs"
+VX_IANA_ENTERPRISE_OID="1.3.6.1.4.1.59817"
+
+# ---------- Helpers ----------
+
+# User input helpers
+
+function get_machine_private_key_password_from_user_input() {
+    local prompt="Enter a private key password (at least 4 characters long): "
+    local validation_error_message="Expected private key password to be at least 4 characters long"
```

Comment:
I wonder if we should generate these automatically, using xkcdpass for 4 dictionary words for example

Reply:
we don't need this passphrase anywhere else right? It's just going to be stored on this disk? So yeah we could generate this as an actually random string that has 256 bits of entropy. Happy to discuss.